r/tests/testthat/test-json.R

```diff
@@ -208,7 +208,7 @@ test_that("Can read json file with nested columns (ARROW-5503)", {
   expect_equal(struct_array$GetFieldByName("ps"), ps)
   expect_equal(struct_array$Flatten(), list(ps, hello))
   expect_equal(
-    as.vector(struct_array),
+    as.data.frame(struct_array),
```

Comment:
Another example of a slight change in our `as.*` API. I think this is better now, but we will need to make sure the change is clearly communicated